airbyte-integrations/connectors/source-gong/manifest.yaml

```
@@ -1352,6 +514,570 @@ schemas:
+                    properties:
+                      count:
+                        type:
+                          - number
+                          - "null"
+                      occurrences:
+                        type:
+                          - array
+                          - "null"
+                        items:
+                          type:
+                            - object
+                            - "null"
+                          properties:
+                            speakerId:
+                              type:
+                                - string
+                                - "null"
+                            startTime:
+                              type:
+                                - number
+                                - "null"
+                      phrase:
+                        type:
+                          - string
+                          - "null"
+      context:
+        description: A list of the agenda of each part of the call.
+        properties:
+          objects:
+            type: array
+            description: List of objects within the external system.
+            items:
+              type: object
+              properties:
+                fields:
+                  type: array
+                  description: array<External object data>.
+                  items:
+                    type: object
+                    properties:
+                      name:
+                        type:
+                          - "null"
+                          - string
+                        description: >-
+                          Field name. For Account supported fields are: Name,
+                          Website, Industry and all the custom fields. For
+                          Opportunity supported fields are Name, LeadSource,
+                          Type, StageName, Probability, Amount, CloseDate and
+                          all the custom fields.
+                      value:
+                        type:
+                          - "null"
+                          - object
+                        description: Field value.
+                objectId:
+                  type:
+                    - "null"
+                    - string
+                  description: Object ID.
+                objectType:
+                  type:
+                    - "null"
+                    - string
+                  description: "Object Type. Allowed: Opportunity, Account"
+                timing:
+                  type:
+                    - "null"
+                    - string
+                  description: "Timing of object. Allowed: Now, TimeOfCall."
+          system:
+            type:
+              - "null"
+              - string
+            description: >-
+              External system name. Allowed: Salesforce, HubSpot,
+              MicrosoftDynamic, Generic.
+      id:
+        type:
+          - "null"
+          - string
```

```suggestion
          - integer
```

Oddly enough, even though this value is being extracted from `metadata.id`, which is a `string`, it is being cast as an `integer` in the final record. This might be a quirk of Jinja interpolation? It's not a big enough concern to block this PR, so I'd say let's just update the expected type here and I can create an issue to look into this.